neo/rawio/blackrockrawio.py

```diff
@@ -495,7 +525,7 @@ def _parse_header(self):
                         chidx_ann['nev_lo_freq_order'] = neuevflt['lo_freq_order'][get_idx]
                         chidx_ann['nev_lo_freq_type'] = flt_type[neuevflt['lo_freq_type'][
                             get_idx]]
-            if self.__nsx_spec[self.nsx_to_load] in ['2.2', '2.3'] and self.__nsx_ext_header:
+            if self.__nsx_spec[self.nsx_to_load[0]] in ['2.2', '2.3'] and self.__nsx_ext_header:
```

Comment:
Is it possible that files are from differents version ?

Reply:
Not if the recording was done properly, but just in case someone mixes up files this might lead to unexpected results, no? So I thinks it would be good to check for this and raise an Error if the files have different file versions.

Reply:
OK I will do this assertion/

Do you also want to test if nev and nsX have the same version ?

Reply:
I guess they also always have to have the same version, since they are recorded simultaneously with the same software, so yes.

Reply:
Done.